composer.json

```diff
@@ -16,16 +16,16 @@
         }
     ],
     "require": {
-        "php": "^7.3",
+        "php": "^7.4 || ^8.0",
         "ext-json": "*",
         "ext-mbstring": "*",
         "avtocod/specs": "~3.70",
         "illuminate/support": "~6.0 || ~7.0 || ~8.0",
-        "ocramius/package-versions": "^1.4"
+        "ocramius/package-versions": "2.1.0"
     },
     "require-dev": {
         "phpunit/phpunit": "~7.5 || ^8.0",
-        "phpstan/phpstan": "~0.12"
+        "phpstan/phpstan": "0.12.99"
```

Comment:
When install PHPStan by mask `~0.12` with lowest dependencies on PHP `8.0`, we have warning & fail:

```bash
> @php ./vendor/bin/phpstan analyze -c ./phpstan.neon.dist --no-progress --ansi
Warning: Undefined array key "PHPStan\Rules\Arrays\DuplicateKeysInLiteralArraysRule" in phar:///src/vendor/phpstan/phpstan/phpstan/vendor/nette/utils/src/Utils/Reflection.php on line 180
Fatal error: Uncaught TypeError: _HumbugBox5f943a942674\Nette\Utils\Reflection::getUseStatements(): Return value must be of type array, null returned in phar:///src/vendor/phpstan/phpstan/phpstan/vendor/nette/utils/src/Utils/Reflection.php:180
```